.vscode/settings.json

```
@@ -6,5 +6,6 @@
   "files.trimTrailingWhitespace": true,
   "typescript.preferences.importModuleSpecifier": "project-relative",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "eslint.validate": ["json"]
```

This is legacy according to https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint .
```suggestion
  "files.eol": "\n"
```